devito/finite_difference.py

```diff
@@ -137,20 +137,27 @@ def first_derivative(*args, **kwargs):
     dim = kwargs.get('dim', x)
     diff = kwargs.get('diff', h)
     order = kwargs.get('order', 1)
-    side = kwargs.get('side', 1)
+    side = kwargs.get('side', "centered")
```

Comment:
I think what @FabioLuporini was asking for was:
```python
left = -1
centered = 0
right = 1
kwargs.get('side', centered)
...
if side == right:
...
```

String comparison definitely should not be going into the codebase